File: DSA/Manav-Rajput/Day3_question_1.java
import java.util.*;

public class PangramChecker {
    public static String checkPangram(String sentence) {
        
        sentence = sentence.toLowerCase();

        HashSet<Character> letters = new HashSet<>();

        // Iterate through each character in the sentence
        for (char c : sentence.toCharArray()) {
            if (Character.isLetter(c)) {
                letters.add(c);
            }
        }

        // Check if the set contains all 26 letters of the alphabet
        if (letters.size() == 26) {
            return "complete";
        } else {
            return "incomplete";
        }
    }

    public static void main(String[] args) {
        String sentence = "The quick brown fox jumps over the lazy dog";
        System.out.println(checkPangram(sentence)); // Output: complete
    }
}

File: DSA/Pragya_Agarwal/Day3_Question1.py
sentence = "The quick brown fox jumps over the lazy dog"
sentence_lower = sentence.lower()
letters = set()

for char in sentence_lower:
    if 'a' <= char <= 'z':
        letters.add(char)
    if len(letters) == 26:
        break

if len(letters) == 26:
    print("complete")
else:
    print("incomplete")
